### DISCUSSION TITLE
Ambient light issues

### DISCUSSION THREAD

**Discussion by Dyya** [cables-gl/cables_docs#292] [! Welcome - readme first!]
About three days ago, ambient light (for all my scenes) stopped working or changed behavior on some devices. Before that, I tested all projects multiple times on all devices without any issues last week. 

Additional details:
1. The ambient light doesn't seem to work on mobile devices (tested on iPhone 11 and 13), 
2. On OS X (Brave, Firefox, everything works perfectly. Safari has the same issue as mobile phones.
3. This setup will cause glitches and sporadic loss of ambient light in Windows 11 (Brave, Firefox). 

[Example scene](https://cables.gl/p/lTFuN7)

**Comment by steam0r** [comment]
hey, we didnt change anything on cables.gl in the past few weeks, we only do this during announced releases actually. did you maybe update your phone/desktop os or anything else that could have changed?

**Reply by Dyya** [reply]
That makes sense. I didn't make any system or software-related updates in the meantime.
Even when I call a 10-day-old backup, it looks broken in the mentioned environments.

**Reply by steam0r** [reply]
hm, what would i be seeing if things are broken? tested this on my m1, monterey 12.4, safari and it seems fine...

**Comment by Dyya** [comment]
Examples with the same scene (Left Safari, right Brave, Monterey 12.4).
<img width="1672" alt="Screenshot 2022-07-12 at 13 06 35" src="https://user-images.githubusercontent.com/8274607/178476407-3f0f1bab-f916-4f19-b815-38b4f3f5c257.png">

<img width="1684" alt="Screenshot 2022-07-12 at 13 06 46" src="https://user-images.githubusercontent.com/8274607/178476397-67d96b47-b63f-4922-be33-dcc286f3da0d.png">
